unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/Resources/Prefabs/CharacterController.prefab

```diff
@@ -373,7 +381,7 @@ MonoBehaviour:
   runningSpeedMultiplier: 0.36
   groundLayers:
     serializedVersion: 2
-    m_Bits: 1025
+    m_Bits: 132097
```

Comment:
Can you add a test to prevent this from accidentally change?

Reply:
Sure, I mean it will change if any other layer is added/removed from the ground layers for the character.

Right now in production with SDK6, GLTF colliders are not defaulted to use `CharacterOnly` and use `Default` layer (that's already in the ground layers of the character controller) because of the old problem of builder scenes deployed with an incorrect state of their GLTFComponent collision property.

With the SDK7 correcting that, GLTF colliders get configured correctly and that's why they have the `CharacterOnly` layer by default (and not considered ground in the character controller as it wasn't needed before).

Anyway I'll code a test to confirm the character detects ground on the now 3 layers considered as ground.

Reply:
done! added test at https://github.com/decentraland/unity-renderer/pull/3743/commits/1a95834569a6fca8d5fb45a9d0b2c44f72c6a17c